### DISCUSSION TITLE
Bagaimana Cara Menampilkan Asset File CSS dan JS dalam Bentuk Inline CSS/JS?

### DISCUSSION THREAD

**Discussion by yuliakbard** [mecha-cms/mecha#182] [Q&A] [answered]
Kang ijin tanya,
`Asset::set('css/index.min.css',20);` ini akan menampilkan css dalam bentuk eksternal css, bagaimana cara menampilkannya dalam bentuk inline css 

```
<style>
.........
</style>
```

untuk penggunaan halaman AMP, yang tidak memperbolehkan eksternal css

**Comment by taufik-nurrohman** [comment, marked as answer]
~~~ php
<?php

$script = <<<JS

/* Kode JavaScript di sini… */

JS;

$style = <<<CSS

/* Kode CSS di sini… */

CSS;

Asset::script($script, 10);

Asset::style($style, 10);
~~~

**Reply by yuliakbard** [reply]
`Asset::set('css/index.min.css',20);` apakah ini gak bisa di konversi jadi inline css kang?

**Reply by taufik-nurrohman** [reply]
Mau bikin versi AMP ya?

~~~ php
if ($path = Asset::path('css/index.min.css')) {
    Asset::style(content($path), 20);
}
~~~

**Reply by yuliakbard** [reply]
``` php
if ($path = Asset::path('css/index.min.css')) {
    Asset::style(content($path), 20);
}
```

kang, gak ada outputnya kang

**Reply by taufik-nurrohman** [reply]
Coba begini saja:

``` php
if ($path = File::exist(__DIR__ . '/asset/css/index.min.css')) {
    Asset::style(content($path), 20);
}
```

**Reply by yuliakbard** [reply]
bisa kang, cuma jadi ada pertanyaan lagi.

kalau di halaman amp itu `<style>` nya ada atribut `amp-custom`

``` html
<style amp-custom>

</style>
```

cara menambahkan attribut amp-customnya gimana kang?

**Reply by taufik-nurrohman** [reply]
Coba cek lagi laman diskusi soal AMP, sudah Saya perbarui di bagian cara konversi elemen secara otomatis.
